src/components/Profile/Profile.js

```diff
@@ -25,10 +26,13 @@ function beforeUpload(file) {
 
 class UserProfile extends Component {
   state = {
-    name: "",
-    email: "",
-    password: "",
+    alertMessage: null,
+    initialName: this.props.authUser.name,
+    initialEmail: this.props.authUser.email,
```

Comment:
initialName and initialEmail States are not required use it directly from props.